### PR TITLE
Final admin changes for v2.0 release

### PR DESCRIPTION
**Pull request by MatthewDaggitt**
Would appreciate someone giving a quick review so we can get this merged in and I can make the release tonight!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Version 2.0-rc1
-===============
+Version 2.0
+===========
 
 The library has been tested using Agda 2.6.4.
```
Comment (Taneb) [inline CHANGELOG.md:4]: This line should also change if we're using a different version in CI

Comment (Taneb) [reply CHANGELOG.md:4]: Also, there's a mention of the version in `doc/installation-guide.md` which should be updated too

Comment (MatthewDaggitt) [reply CHANGELOG.md]: Thanks, fixed!

File: .github/workflows/ci-ubuntu.yml [hunk 2]
```diff
@@ -79,7 +79,7 @@ jobs:
             echo "AGDA_HTML_DIR=html/experimental" >> $GITHUB_ENV
           else
             # Pick Agda version for master
-            echo "AGDA_COMMIT=tags/v2.6.4" >> $GITHUB_ENV;
+            echo "AGDA_COMMIT=tags/v2.6.4.1" >> $GITHUB_ENV;
```
Comment (jamesmckinna) [inline .github/workflows/ci-ubuntu.yml:82]: Do we have a philosophy on major/minor versions of Agda for CI? Perhaps the question really for Agda itself: which versions are regarded as 'stable'/'authoritative' for subsequent development?

**Comment by jamesmckinna** [comment]
Hmmm, having trouble finishing my review... but I approve!

**Review by jamesmckinna — APPROVED** [review]

**Comment by jamesmckinna** [comment]
If you time zone permits @MatthewDaggitt please do merge, otherwise I will in the next hour or so.

**Pull request merged**